thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,37 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetImageDiff(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetImageDiff")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	opts1 := ImageDiffOptions{
+		AMI:   "ami-029ab576546a58916",
+		AMI2:  "ami-02b25f680ad574d33",
+		Page:  0,
+		Limit: 10,
+	}
+	result1, err := c.getImageDiff(ctx, opts1)
```

Comment:
Like before, overwrite `result` instead
```suggestion
	result, err := c.getImageDiff(ctx, opts)
```